File: src/main/java/ru/citeck/idea/codeinsight/forms/components/EcosSelect.java
package ru.citeck.idea.codeinsight.forms.components;

import lombok.Getter;

import java.util.Map;

@Getter
public class EcosSelect extends PropertyComponent {

    public static final String TYPE = "ecosSelect";

    private final Map<String, String> data = Map.of("url", "/citeck/ecos/records/query");
    private final Boolean lazyLoad = false;

    @Override
    public String getType() {
        return TYPE;
    }

}

File: src/main/kotlin/ru/citeck/idea/artifacts/action/codeinsight/forms/components/EcosSelect.kt
package ru.citeck.idea.artifacts.action.codeinsight.forms.components

class EcosSelect : PropertyComponent() {

    companion object {
        const val TYPE: String = "ecosSelect"
    }

    val data = mapOf("url" to "/citeck/ecos/records/query")
    val lazyLoad = false

    override fun getType(): String {
        return TYPE
    }
}
